js/components/highchart/index.js

```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+const ReactHighcharts = require('react-highcharts');
+
+export default class MyHighchart extends Component {
+  constructor (...args) {
+    super(...args);
+
+    this.state = { 
```

Comment:
you don't need to save all config in state. In state you must store only values what you will change